src/types/Status.ts

```diff
@@ -0,0 +1 @@
+export type Status = 'all' | 'active' | 'completed';
```

Comment:
You can make it an enum instead of a type. This way you will be able to use it in code like Status.ALL instead of strings